test/run_envoy_tests.sh

```diff
@@ -61,10 +61,16 @@ then
   exit 0
 fi
 
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.
+HOT_RESTART_JSON="$TEST_SRCDIR"/test/config/integration/hot_restart.json
+cat "$TEST_TMPDIR"/test/config/integration/server.json |
```

Comment:
This used to test socket passing, but I don't think it actually does anymore since each listener now binds to a random port? Is there any way to restore the old behavior that is sane? At least can we put in TODO to restore the old test behavior?